tesseract/tesseract_planning/tesseract_process_managers/src/process_planning_server.cpp

```diff
@@ -0,0 +1,429 @@
+/**
+ * @file process_planning_server.cpp
+ * @brief A process planning server with a default set of process planners
+ *
+ * @author Levi Armstrong
+ * @date August 18, 2020
+ * @version TODO
+ * @bug No known bugs
+ *
+ * @copyright Copyright (c) 2020, Southwest Research Institute
+ *
+ * @par License
+ * Software License Agreement (Apache License)
+ * @par
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * @par
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <tesseract_common/macros.h>
+TESSERACT_COMMON_IGNORE_WARNINGS_PUSH
+#include <console_bridge/console.h>
+TESSERACT_COMMON_IGNORE_WARNINGS_POP
+
+#include <tesseract_process_managers/process_planning_server.h>
+#include <tesseract_process_managers/debug_observer.h>
+#include <tesseract_process_managers/taskflow_generators/cartesian_taskflow.h>
+#include <tesseract_process_managers/taskflow_generators/descartes_taskflow.h>
+#include <tesseract_process_managers/taskflow_generators/freespace_taskflow.h>
+#include <tesseract_process_managers/taskflow_generators/ompl_taskflow.h>
+#include <tesseract_process_managers/taskflow_generators/trajopt_taskflow.h>
+#include <tesseract_process_managers/taskflow_generators/raster_taskflow.h>
+#include <tesseract_process_managers/taskflow_generators/raster_global_taskflow.h>
+#include <tesseract_process_managers/taskflow_generators/raster_only_taskflow.h>
+#include <tesseract_process_managers/taskflow_generators/raster_only_global_taskflow.h>
+#include <tesseract_process_managers/taskflow_generators/raster_dt_taskflow.h>
+#include <tesseract_process_managers/taskflow_generators/raster_waad_taskflow.h>
+#include <tesseract_process_managers/taskflow_generators/raster_waad_dt_taskflow.h>
+
+#include <tesseract_motion_planners/descartes/profile/descartes_profile.h>
+#include <tesseract_motion_planners/trajopt/profile/trajopt_profile.h>
+#include <tesseract_motion_planners/ompl/profile/ompl_profile.h>
+#include <tesseract_motion_planners/descartes/profile/descartes_profile.h>
+#include <tesseract_motion_planners/simple/profile/simple_planner_profile.h>
+
+#include <tesseract_command_language/utils/utils.h>
+
+namespace tesseract_planning
+{
+TaskflowGenerator::UPtr createTrajOptGenerator()
+{
+  TrajOptTaskflowParams params;
+  return std::make_unique<TrajOptTaskflow>(params);
+}
+
+TaskflowGenerator::UPtr createOMPLGenerator()
+{
+  OMPLTaskflowParams params;
+  return std::make_unique<OMPLTaskflow>(params);
+}
```

Comment:
I think I would rather put these in a separate file so that the core planning server code isn't way at the bottom.

Reply:
Moved.